pkgs/applications/networking/instant-messengers/telegram/tdesktop/tg_owt.nix

```diff
@@ -24,15 +24,15 @@ stdenv.mkDerivation {
 
   buildInputs = [
     libjpeg openssl libopus ffmpeg alsa-lib libpulseaudio protobuf
-    xorg.libX11 libXtst libXcomposite libXdamage libXext libXrender libXrandr
+    openh264 usrsctp libevent libvpx
+    libX11 libXtst libXcomposite libXdamage libXext libXrender libXrandr libXi
     glib abseil-cpp pcre util-linuxMinimal libselinux libsepol pipewire
-    libXi
   ];
 
-  cmakeFlags = [
-    # Building as a shared library isn't officially supported and currently broken:
-    "-DBUILD_SHARED_LIBS=OFF"
```

Comment:
That's not a good idea, tg_owt has no stable ABI and share build is community supported (breaks after each big update and requires a long time to be fixed)

Reply:
Fixed.